pkgs/applications/misc/calibre/default.nix

```diff
@@ -88,6 +88,15 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
+  # Remove some references to shrink the closure size. This reference (as of
+  # 2018-11-06) was a single string like the following:
+  #   /nix/store/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-podofo-0.9.6-dev/include/podofo/base/PdfVariant.h
+  preFixup = ''
+    remove-references-to -t ${podofo.dev} $out/lib/calibre/calibre/plugins/podofo.so
+  '';
+
+  disallowedReferences = [ podofo.dev ];
```

Comment:
Is podofo.out still there?

Reply:
Nope, just `podofo.lib`; the `.out` output is all binaries and manpages. Given that, I didn't think it was worth adding it to the list of `disallowedReferences`, but happy to change if you think it's worth it?

Reply:
Ok looks good then